### PR TITLE
Add fixes for PHP 8.2

### PR DESCRIPTION
**Pull request by srtfisher**
Add fixes for PHP 8.2 dynamic properties.

### CODE REVIEW

**Comment by EAGLESWhit** [comment]
Please review this soon. WPEngine is having everyone go to PHP 8.2 at the end of October.

"WP Engine’s PHP 8.0 will be reaching end of life on October 31. Prepare your websites for the PHP 8.2 upgrade to experience increased security and performance." - WPEngine

File: common/php/class-module.php [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 if ( !class_exists( 'EF_Module' ) ) {
 
+#[\AllowDynamicProperties]
```
Comment (rebeccahum) [inline common/php/class-module.php:10]: Is this attribute necessary since I see that the properties are later being defined?

File: edit_flow.php [hunk 2]
```diff
@@ -53,6 +53,7 @@ function _ef_print_php_version_admin_notice() {
 define( 'EDIT_FLOW_SETTINGS_PAGE' , add_query_arg( 'page', 'ef-settings', get_admin_url( null, 'admin.php' ) ) );
 
 // Core class
+#[\AllowDynamicProperties]
```
Comment (rebeccahum) [inline edit_flow.php:56]: Same as above ^^.

**Review by rebeccahum — CHANGES_REQUESTED** [review]

**Comment by rebeccahum** [comment]
Closing in favour of #700

**Pull request closed without merging**